.gitignore

```diff
@@ -234,3 +234,5 @@ win_configure/projects.VS2008/pbs_ifl_wrap.c
 src/cmds/scripts/modulefile
 src/cmds/scripts/pbs.service
 
+# Generated by make dist
+*.tar.gz
```

Comment:
If you're going to ignore archive files, I suggest you add a few more...
*.tar
*.tar.bz
*.tgz
*.zip
*.cpio
*.rpm
*.deb

There may be many others, but you get the idea.